client/src/components/spinner.component.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { describe, expect, it } from 'vitest';
+import { AstrosatSpinner } from './spinner.component';
+import { render, screen } from '~/test/utils';
+describe('spinner', () => {
```

Comment:
Use whitespace to separate out your tests, so it makes it more readable, think about natural language, we all use spaces to separate words, paragraphs, chapters etc, code is no different.

* new line between imports of different types
* new line between imports and `describe` blocks
* new line between `it` blocks
* new line between render, doing actions and expects

If code is logically different, put some whitespace between them.